src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+public class Task {
+    private String description;
+    private boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public String toString() {
+        return "[" + getStatusIcon() + "] " + description;
+    }
+
+    private String getStatusIcon() {
+        return (isDone ? "X" : " "); // return X if task is done, and space otherwise
```

Comment:
It is recommended to place a comment on top of the statement it is referring to for clarity purpose.

Reply:
comments should be indented relative to their position in the code